app/js/states/policies/views/list-policies.jade

```diff
@@ -0,0 +1,43 @@
+.policies-page
+  page-header(title="Policies", icon="fa-gavel")
+
+  .content-wrap
+    section.table-utilities
+      .row
+        .col-xs-7.col-sm-5
+          search-box(placeholder="{{'Search policies' | translate}}", on-search='search(model)', throttle='1000', ng-model='searchText')
+    .row.row-short
+      .col-md-12
+        table.table.ng-table
+          thead
+            tr
+              th.sortable(ng-class="sorter.getSortClass('policy_name')", ng-click="sorter.sort('policy_name')", translate) Policy
+              th.nowrap.sortable(ng-class="sorter.getSortClass('systems_pass')", ng-click="sorter.sort('systems_pass')", translate) Systems Passing
```

Comment:
![screenshot_20171221_164345](https://user-images.githubusercontent.com/167352/34262974-373147b2-e66e-11e7-8717-e9855dbee66d.png)

The sorting indication does not match (counts descending while the indicator shows ascending)

Reply:
This is fixed with @ryelo's unportal fixes
#346